packages/languages/src/browser/language-client-contribution.ts

```diff
@@ -98,8 +98,9 @@ export abstract class BaseLanguageClientContribution implements LanguageClientCo
             commands: this,
             documentSelector: this.documentSelector,
             synchronize: { fileEvents },
-            initializationFailedHandler: () => {
-                this.messageService.error("Failed to start language server '" + this.name + "'.");
+            initializationFailedHandler: (err) => {
+                let detail = err instanceof Error ? `: ${err.message}` : '.';
```

Comment:
Please change `let` to `const`.

Reply:
thought we have linting errors for it, surprisingly build does not fail

Reply:
Oh well, good point. Does it worth a follow-up task?

Reply:
yes, would be nice to fix it